src/map/script_constants.hpp

```diff
@@ -813,8 +813,10 @@
 	export_constant(EQP_ACC_RL);
 	export_constant(EQP_SHADOW_ACC_RL);
 
+	export_constant2("EQP_Both_Hand", (EQP_HAND_R|EQP_HAND_L));
```

Comment:
Do not ask me why, but there are some additional EQP defines inside pc.hpp

This should use: EQP_ARMS